<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTest</class>
 <widget class="QDialog" name="ConfigTest">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration des tests</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QListView" name="listViewConfigTest"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Paramètres de test actuel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="2" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxParametre">
         <property name="text">
          <string>Paramètre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxValeur">
         <property name="text">
          <string>Valeur</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBoxIntervalle">
         <item>
          <property name="text">
           <string>min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>heure</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxIntervalle">
         <property name="text">
          <string>Intervalle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxIntervalle"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxFirstOperande">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="maxVisibleItems">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>Tous</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxOperation">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Sup</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inf</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sup ou =</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inf ou =</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sup et Inf</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sans</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nouveau paramètre de test</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxValeur">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="comboBoxType">
           <item>
            <property name="text">
             <string>Remarque</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Correction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Invalide</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBoxParametre">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="maxVisibleItems">
            <number>5</number>
           </property>
           <item>
            <property name="text">
             <string>Tous</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Type :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QTableWidget" name="tableValeurstest">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>115</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>115</number>
           </attribute>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Liste des valeurs du test</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonAjouterParametre">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ajouter paramètre</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonModifierParametre">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Modifier paramètre</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonAnnuler">
           <property name="text">
            <string>Annuler</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigTest</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigTest</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
